observability changes to application configs

lake-of-dreams opened the pull request.

markxnelson commented on cloudbank-v4/account/src/main/resources/application.yaml:
```diff
@@ -76,10 +87,21 @@ management:
   level:
     root: INFO
-    com.example: INFO
+    com.example: DEBUG
```
do we really need debug on com.example?  seems a bit high?

lake-of-dreams replied:
there are not enough INFO logs corresponding to traces, trying to see how can optimize

lake-of-dreams commented:
1. Created a `common` module that includes a `LoggingFilterConfig` config bean to log some details about request/response to correlate with traceIds.
2. Moved common configuration from `application.yaml` of all modules to the common module and created a `spring.config.import` in sibling modules.
3. Removed `DEBUG` logging to prevent high verbosity, now sufficient info is being printed with traces.

markxnelson reviewed (approved):
we will need matching updates to the lab guides too fyi @andytael
